l1-contracts/src/interfaces/IInbox.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
why do you have 2 IInbox interfaces?

Reply:
Seems this was left over from the merge, as your previous pr had it in a different location it did not cop they were really the same file.
This has been addressed and they have been removed

Reply:
Ah merge issue they existed in this branch in a different location to where they were merged in in yours, I've removed